quickwit-cli/src/lib.rs

```diff
@@ -129,3 +134,30 @@ mod tests {
         Ok(())
     }
 }
+
+/// A struct to print data on the standard output.
+pub struct Printer<'a> {
```

Comment:
Do we still need this?

Reply:
Yes, we currently use it in the indexer, I just moved it over to reuse. 
I will move it back to where it was.